synapse/models/orgs.py

```diff
@@ -76,8 +76,56 @@ def getModelDefs(self):
                 ('ou:campaign', ('guid', {}), {
                     'doc': 'Represents an orgs activity in pursuit of a goal.',
                 }),
+                ('ou:id:type', ('guid', {}), {
+                    'doc': 'A type of id number issued by an org.',
+                }),
+                ('ou:id:value', ('str', {'strip': True}), {
```

Comment:
do we care about case here? ``DL1234`` vs ``dl1324``

Reply:
Unfortunately, we can't predict whether an org will create an ID that is case sensitive.  Lets discuss.

Reply:
Gonna leave it case sensitive.  Too many potential issues with normalization. :D